puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_kafo_pull_request.yaml

```diff
@@ -25,16 +25,18 @@
           - '2.2'
           - '2.3'
           - '2.4'
+          - '2.5'
       - axis:
           type: user-defined
           name: puppet
           values:
           - '4.10'
```

Comment:
this should get deleted now?

Reply:
We still support Puppet 4.10 in Kafo 3. I'm also not sure how well the Puppet 5 gem works on Ruby 2.0 and I'm not sure how easily we can test cross Ruby versions.